nym-vpn-app/src-tauri/src/commands/account.rs

```diff
@@ -80,7 +88,7 @@ pub async fn account_links(
     locale: String,
 ) -> Result<AccountLinks, BackendError> {
     grpc.account_links(&locale).await.map_err(|e| {
-        error!("failed to get account link: {e}");
+        warn!("failed to get account link: {e}");
```

Comment:
Still we want this as error.

Actually, better to remove this log as the error is already logged in the grpc client method (`grpc.account_links`).